src/client/telemetry/index.ts

```diff
@@ -973,6 +973,11 @@ export interface IEventNamePropertyMapping {
      * Total time taken to get the preferred kernel for notebook.
      */
     [Telemetry.GetPreferredKernelPerf]: undefined | never;
+    [Telemetry.PreferredKernel]: {
```

Comment:
Could we have a comment about what this event represents and when it fires please?

Reply:
Oops, forgot

Reply:
Thanks, done.